.github/workflows/cppcheck-daily.yaml

```diff
@@ -16,20 +16,12 @@ jobs:
       - name: Install dependencies
         run: |
           sudo apt-get update
-          sudo apt-get install -y build-essential cmake git libpcre3-dev
+          sudo apt-get install -y git
```

Comment:
Maybe, we can delete whole of this step

Reply:
Agreed :+1:

Reply:
@HansRobo 
Do you think we can skip even `sudo apt-get update` ?
In other words, all `Install dependencies` steps

Reply:
Yes, we can.
My opening statement means the deletion of whole of  `Install dependencies` steps.